src/ucp/core/ucp_request.c

```diff
@@ -689,16 +689,15 @@ void ucp_request_send_state_ff(ucp_request_t *req, ucs_status_t status)
                     ucp_wireup_msg_str(req->send.wireup.type));
         ucs_free(req->send.buffer);
         ucp_request_mem_free(req);
-    } else if (req->send.state.uct_comp.func == ucp_ep_flush_completion) {
-        ucp_ep_flush_request_ff(req, status);
```

Comment:
is there need to change order, or leftover?

Reply:
restored the previous order, I found that this order doesn't make sense